src/firmware/app/control/trajectory_planner_test.cpp

```diff
@@ -1090,12 +1090,26 @@ TEST_F(TrajectoryPlannerTest,
         app_trajectory_planner_generateConstantParameterizationPositionTrajectory(
             path_parameters, &position_trajectory);
     EXPECT_EQ(status, OK);
+
+    float max_orientation = 0.0f;
+    for (unsigned int i = 0; i < path_parameters.num_elements - 1; i++)
+    {
+        // orientation should always be increasing
+        EXPECT_GE(position_trajectory.orientation[i], max_orientation);
+        if (position_trajectory.orientation[i] > max_orientation)
+        {
+            max_orientation = position_trajectory.orientation[i];
+        }
+    }
+    EXPECT_GT(max_orientation, 900.0f);
```

Comment:
We probably want to make sure this is upper-bounded as well? Or is that covered in another test?

Reply:
added an upper bound